conf/messages.en

```diff
@@ -108,7 +109,7 @@ ukVatRegisteredPostcode.checkYourAnswersLabel = UK VAT registration postcode
 ukVatRegisteredPostcode.error.required = Enter the postcode where your business is registered for UK VAT
 ukVatRegisteredPostcode.error.invalid = Enter a valid postcode
 ukVatRegisteredPostcode.error.length = Postcode must be 9 characters or less
-ukVatRegisteredPostcode.change.hidden = the postcode where your business is registered for UK VAT
+ukVatRegisteredPostcode.change.hidden = Change the postcode where your business is registered for UK VAT
```

Comment:
I don't think we need the word "change" at the start of any of the hidden text.  This gets rendered as visually hidden text in the "Change" link, so repeating it here would mean a screen reader would read the word twice